plugin/plugins/testPlugin/__init__.py

```
@@ -1,16 +1,572 @@
+            watch_source = "testPlugin.watch.demo"
+            watch_list = self.ctx.bus.messages.get(
+                plugin_id=plugin_id or None,
+                max_count=max_count,
+                priority_min=pri_opt,
+                timeout=timeout,
+            ).filter(source=watch_source, strict=True)
+            watcher = watch_list.watch(self.ctx).start()
+            try:
+                self.file_logger.info("[messages_watch] watcher started: source={}", watch_source)
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)